website/docs/r/policy_lb_icmp_monitor_profile.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Load Balancer"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_lb_icmp_monitor_profile"
+description: A resource to configure a LBIcmpMonitorProfile.
+---
+
+# nsxt_policy_lb_icmp_monitor_profile
+
+This resource provides a method for the management of a LBIcmpMonitorProfile.
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_lb_icmp_monitor_profile" "test" {
+  display_name = "test"
+  description  = "Terraform provisioned LBIcmpMonitorProfile"
+  data_length  = 2
+  fall_count   = 2
+  interval     = 2
+  monitor_port = 8080
```

Comment:
`monitor_port` needs to be removed for `icmp` profile, here and below in attribute description. Other than that LGTM, thanks

Reply:
Thanks Anna, monitor port has been removed